cmake/modules/AddSwift.cmake

```diff
@@ -1453,6 +1458,11 @@ function(add_swift_library name)
         continue()
       endif()
 
+      # TODO Support SwiftPrivate on Windows (SR-6489)
```

Comment:
Can you add to the comment here to describe the problem. I think just saying something like this would be fine:

```
TODO: Currently SwiftPrivate runs into linker problems on Windows. See SR-6489.
```

Reply:
Done.